### PR TITLE
Added sorting plugin to submissions table for staff, Now you can sort each column by clicking header

### PR DESCRIPTION
**Pull request by amir-qayyum-khan**
In this PR i added sort table plugin to SGA in submissions section for staff, Now we can sort submissions by any column i.e by clicking column header **"Username, Name, FileName etc"**

cc @pdpinch 
Issue https://github.com/mitodl/edx-sga/issues/101

screen shot 2015-06-28 at 11 46 46 pm

screen shot 2015-06-28 at 11 46 51 pm

### CODE REVIEW

File: edx_sga/static/js/src/edx_sga.js [hunk 1]
```diff
@@ -152,6 +152,10 @@ function StaffGradedAssignmentXBlock(runtime, element) {
 
                 updateChangeEvent(fileUpload);
             });
+            $("#submissions").tablesorter();
+            $("#submissions").trigger("update");
+            var sorting = [[0,0]];
+            $("#submissions").trigger("sorton",[sorting]);
```
Comment (amir-qayyum-khan) [inline edx_sga/static/js/src/edx_sga.js]: Default sorting by column **UserName**

**Comment by pdpinch** [comment]
What's the default sort?

**Comment by amir-qayyum-khan** [comment]
Name

@pdpinch

File: edx_sga/sga.py [hunk 2]
```diff
@@ -215,6 +215,7 @@ def student_view(self, context=None):
         )
         fragment.add_css(_resource("static/css/edx_sga.css"))
         fragment.add_javascript(_resource("static/js/src/edx_sga.js"))
+        fragment.add_javascript(_resource("static/js/src/jquery.tablesorter.min.js"))
```
Comment (pwilkins) [inline edx_sga/sga.py]: I have a question for my own information.  I see you are including the minimized version of this library.  Doesn't edX already manage compressing javascript?  Or is it a best practice to use minimized versions of javascript libraries?

File: edx_sga/templates/staff_graded_assignment/show.html [hunk 3]
```diff
@@ -48,16 +48,19 @@
...
-        <th>Uploaded</th>
-        <th>Grade</th>
-        <th>Instructor's comments</th>
-        <th>Annotated</th>
+        <th class="header">Username <i class="icon fa fa-sort"/></th>
+        <th class="header">Name <i class="icon fa fa-sort"/></th>
+        <th class="header">Filename <i class="icon fa fa-sort"/></th>
+        <th class="header">Uploaded <i class="icon fa fa-sort"/></th>
+        <th class="header">Grade <i class="icon fa fa-sort"/></th>
+        <th class="header">Instructor's comments <i class="icon fa fa-sort"/></th>
```
Comment (pwilkins) [inline edx_sga/templates/staff_graded_assignment/show.html]: Just an observation: At first I wondered why anyone would sort on the comment field, but then I considered that an instructor could preface comments with a tag or their initials, making a sorted comment field useful.

File: edx_sga/sga.py (hunk 2 above)
Comment (pdpinch) [inline edx_sga/sga.py]: I'm not sure if the javascript build pipeline includes XBlock dependencies. @bdero do you know?

**Comment by pdpinch** [comment]
This needs to be tested in Cypress.

**Comment by jetej** [comment]
This seems to work ok for most of the sorting options. I noticed that I wasn't able to sort by "Filename", "Uploaded", or the "Annotated" file name.

**Comment by pdpinch** [comment]
@amir-qayyum-khan it would be really useful to be able to sort by (date) `uploaded`. Do you know why that isn't working?

For fields that don't sort, we should remove the sort icon.

**Comment by amir-qayyum-khan** [comment]
@pdpinch @jetej there was issue with alphanumeric string , i fixed this,  now all columns are working.
Thanks,

**Comment by jetej** [comment]
Works great!

**Comment by amir-qayyum-khan** [comment]
:tada:

**Pull request merged**